nifi-nar-bundles/nifi-site-to-site-reporting-bundle/nifi-site-to-site-reporting-task/src/main/resources/docs/org.apache.nifi.reporting.SiteToSiteStatusReportingTask/additionalDetails.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+    <!--
+      Licensed to the Apache Software Foundation (ASF) under one or more
+      contributor license agreements.  See the NOTICE file distributed with
+      this work for additional information regarding copyright ownership.
+      The ASF licenses this file to You under the Apache License, Version 2.0
+      (the "License"); you may not use this file except in compliance with
+      the License.  You may obtain a copy of the License at
+          http://www.apache.org/licenses/LICENSE-2.0
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+    -->
+    <head>
+        <meta charset="utf-8" />
+        <title>SiteToSiteStatusReportingTask</title>
+
+        <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css" />
+    </head>
+
+    <body>
+    	<p>
+    		The Site-to-Site Bulletin Reporting Task allows the user to publish Status events using the Site To Site protocol. 
+    		The component type and name filter regexes form a union: only components matching both regexes will be reported. 
+    		However, all process groups are recursively searched for matching components, regardless of whether the process 
+    		group matches the component filters.
+    	</p>
+    	
+    	<h2>Record writer</h2>
+   	
+    	<p>
+    		The user can define a Record Writer and directly specify the output format and data with the assumption that the 
+    		input schema is the following:
+    	</p>
+
+		<pre>
+			<code>
+{
+  "type" : "record",
+  "name" : "status",
+  "namespace" : "status",
+  "fields" : [
+    // common fields for all components
+	{ "name" : "statusId", "type" : "string"},
+	{ "name" : "timestampMillis", "type": { "type": "long", "logicalType": "timestamp-millis" } },
+	{ "name" : "timestamp", "type" : "string"},
+	{ "name" : "actorHostname", "type" : "string"},
+	{ "name" : "componentType", "type" : "string"},
+	{ "name" : "componentName", "type" : "string"},
+	{ "name" : "parentId", "type" : "string"},
```

Comment:
Currently in https://github.com/apache/nifi/pull/2663/files#diff-f1f37886c50fa0946558aed14f835e27R147 it is sending in null for the parentId all the time. I think we should determine the parentId if possible, and (at least) make the field in the schema nullable.

Reply:
That's not what I'm seeing: only the status of the root process group does have a null parentId which makes sense. But I do see the correct parentId on the other components. I updated the schema/doc to allow the null value though.